perun-cli/Perun/beans/HashedGenData.pm

```diff
@@ -124,11 +124,32 @@ sub getResourceIds ()
 	return @resourceIds;
 }
 
+sub getVosIds ()
+{
+	my $self = shift;
+	my @vosIds = ();
+	my $facilityId = $self->getFacilityId;
+	foreach my $resourceId (sort keys %{$self->{_hierarchy}->{$facilityId}->{c}}) {
```

Comment:
You could probably call getResourceIds instead.

Reply:
Done.